### PR TITLE
Create आचार संहिता.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
## Checklist

- [x] I ran `npm test` locally and it passed without errors.

- [x] I only edited the `_data/pixels.json` file.

- [x] I entered the `username` in the `pixels.json` that I'm also using to create this pull request.
- [x] I acknowledge that all my contributions will be made under the project's [license](../../LICENSE.md).

### CODE REVIEW

**Comment by athif23** [comment]
I don't know what you are adding here. It looks like a translated documentation, if so, I think you need to create folder in [`docs`](https://github.com/twilio-labs/open-pixel-art/tree/master/docs) folder with the name of the language. You can see the example in [`docs/es`](https://github.com/twilio-labs/open-pixel-art/tree/master/docs/es) folder.

**Comment by sakshamtaneja21** [comment]
ok got it 
ill create a folder then do

File: आचार संहिता.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# योगदानकर्ता आचरण संहिता
+
+¿Español? Puedes encontrar nuestro Código de Conducta traducido en [este enlace] (डॉक्स / तों / CODE_OF_CONDUCT.md)।
```
Comment (dkundel) [inline आचार संहिता.md]: Thanks for the contribution! Could you please add a link to the English one and specify it as the source Code of Conduct? Basically write:

```
This version of the Code of Conduct is a translation for your convenience. If this version and the English original differ, the English one will be applied. 
```

Can you place that at the top in both languages?

**Comment by sakshamtaneja21** [comment]
hello 
can i update it now

**Pull request closed without merging**